pkg/nfd-master/nfd-master.go

```diff
@@ -990,6 +1001,9 @@ func (m *nfdMaster) processNodeFeatureRule(nodeName string, features *nfdv1alpha
 			features.InsertAttributeFeatures(nfdv1alpha1.RuleBackrefDomain, nfdv1alpha1.RuleBackrefFeature, ruleOut.Vars)
 		}
 	}
+	processingTime := time.Since(processStart)
+	crdProcessingTime.Set(float64(processingTime))
+	klog.V(2).InfoS("processed NodeFeatureRule resources", "processed", len(ruleSpecs), "processing time", processingTime)
```

Comment:
Let's try to follow k8s conventions/recommendations, ref https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

Use lowerCamelCase for the field names. `duration` instead of "processing time", let's add node name for context. If we want to log the number of ruleSpecs let's use e.g. `objectCount`.

```suggestion
	klog.V(2).InfoS("processed NodeFeatureRule objects", "nodeName", nodeName, "objectCount", len(ruleSpecs), "duration", processingTime)
```

Feel free to improve or suggest smth else